pkg/web/leaderboard.go

```diff
@@ -43,7 +52,14 @@ func (s *Server) handleGetLeaderboard(w http.ResponseWriter, r *http.Request) er
 			return fmt.Errorf("error rendering quote: %w", err)
 		}
 
+		authorIDs := make([]string, len(quote.Authors))
+		for i, author := range quote.Authors {
+			authorIDs[i] = author.ID
+		}
+		authorNames, err := s.resolveAuthorIDs("<@" + strings.Join(authorIDs, ">, <@") + ">")
```

Comment:
Rather than doing the join this way, could you create mentions in the loop and join those mentions?

```suggestion
		authorMentions := make([]string, len(quote.Authors))
		for i, author := range quote.Authors {
			authorMentions[i] = fmt.Sprintf("<@%s>", author.ID)
		}
		authorNames, err := s.resolveAuthorIDs(strings.Join(authorMentions, ", "))
```